packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -228,7 +228,7 @@ const DragAndDropElement = ({
             role="button"
             tabIndex={0}
             aria-label={formatMessage({
-              id: getTranslation('components.DragHandle-label'),
+              id: getTranslation('content-manager.components.DragHandle-label'),
```

Comment:
this is incorrect, the `getTranslation` function appends `content-manager` to these strings.

Reply:
Oh, where can I detect this ?

Reply:
I'm guessing that prefix append is broken then as I was looking at this with him in Discord and setting the keys to override doesn't do anything, it always takes the static default value.

Reply:
I don't know about `content-manager.components.DragHandle-label` but `content-manager.containers.ListPage.displayedFields` at least was indeed stuck with the default value.